docs/modules/ROOT/pages/explanations/decisions/postgres-monitoring.adoc

```diff
@@ -0,0 +1,109 @@
+= PostgreSQL Managed by VSHN Monitoring
+
+== Problem
+
+We need to provide PostgreSQL Managed by VSHN monitoring checks:
+
+* Availability check
+
+== Requirements for availability check
+
+* PostgreSQL is accessible on network level
+* it's possible to login to PostgreSQL
+* it's possible to execute queries 
+
+== Solutions
+
+PostgreSQL managed by Stackgres:
+
+* it doesn't reuse generated connection string which our users will use, which could lead to situation where our monitoring is green, but users can't login (issues with Service)
+* it's sidecar, issue with exporter would require pod restarting
+* prometheusAutobind from SGCluster doesn't work, so I have a fear that whole monitoring isn't really well tested
+
+PostgreSQL managed by VSHN:
+
+* the same functionalities as above
+* solves issues presented above
```

Comment:
Again it's not really clear what this solution would include.

Where would this exporter run? In the customer namespace? In the database namespace? In a separate namespace?